<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton0">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EqualPushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MinusPushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DotPushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackPushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>⌦</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MulPushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ModlusPushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DivPushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string> ≡</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Standard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PlusPushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
